frontend/src/components/HiddenParticipantsTile/HiddenParticipantsTile.tsx

```diff
@@ -23,17 +24,21 @@ export type HiddenParticipantsTileProps = {
 const HiddenParticipantsTile = ({
   box,
   hiddenSubscribers,
-  handleClick,
 }: HiddenParticipantsTileProps): ReactElement => {
+  const { toggleParticipantList } = useSessionContext();
+  const { meetingRoomSettings } = useConfigContext();
+  const { showParticipantList } = meetingRoomSettings;
   const { height, width } = box;
   const diameter = Math.min(height, width) * 0.38;
   return (
     <button
       id="hidden-participants"
       data-testid="hidden-participants"
-      className="absolute m-1 flex cursor-pointer items-center justify-center rounded-xl bg-notVeryGray-100 transition-colors hover:bg-[rgb(76,80,82)]"
+      className={`absolute m-1 flex items-center justify-center rounded-xl bg-notVeryGray-100 transition-colors ${
+        showParticipantList ? 'cursor-pointer hover:bg-[rgb(76,80,82)]' : 'cursor-default'
+      }`}
       style={getBoxStyle(box)}
-      onClick={handleClick}
+      onClick={showParticipantList ? toggleParticipantList : () => {}}
```

Comment:
If we're not showing the participant list button, we shouldn't display the list when clicking this tile, either.